workflow/automation/execution_scripts/add_to_mgmt_queue.py

```diff
@@ -6,6 +6,14 @@
 import qcore.constants as const
 from workflow.automation.lib.shared_automated_workflow import add_to_queue
 
+
+def convert_time(time: str):
```

Comment:
Maybe calling it `datestr_to_timestamp`?

Or at least having a docstring with a couple of examples.